app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt

```diff
@@ -0,0 +1,3 @@
+package com.nilhcem.blenamebadge.device.model
+
+data class BitmapDataToSend(val messages: List<BitmapMessage>)
```

Comment:
Why's this needed? Why not just use List<BitmapMessage> wherever required?

Reply:
I tried to follow the same pattern which was originally being used in sending text to the device

Reply:
@iamareebjamal I've Updated the PR with the new Layout changes, have a look

Reply:
@iamareebjamal btw sir, you can find the similar class for text here https://github.com/fossasia/badge-magic-android/blob/development/app/src/main/java/com/nilhcem/blenamebadge/device/model/DataToSend.kt